com.unity.visualeffectgraph/Editor/Models/Contexts/VFXContext.cs

```diff
@@ -611,5 +611,24 @@ public override void CheckGraphBeforeImport()
             foreach (var block in children)
                 block.CheckGraphBeforeImport();
         }
+
+        //TODO: Register all the contexts that have issues when transfering settings (in ConvertContext() )
+        protected virtual IEnumerable<string> untransferableSettings
```

Comment:
Not sure about this mechanism? How can we know a setting can be transferred or not from the dest only? Shouldnt it take in consideration source as well?
Why not just transfer settings in dst as soon as the source has one that matches name and type?

Reply:
> Why not just transfer settings in dst as soon as the source has one that matches name and type?
That is already checked in ConvertContext(). The case I'm trying to solve here is when the destination and the source have the setting, but the destination setting needs to have certain value. For e.g. (not added in this PR), Lit Sphere has a blendMode set always to Opaque.

Reply:
Ok got it